src/query/functions/src/scalars/decimal/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "databend-functions-scalar-decimal-utils"
```

Comment:
name should be `databend-functions-scalar-decimal`

Reply:
Done. and keep the register order.

![image](https://github.com/user-attachments/assets/3dabd87b-f782-4be7-9fd6-4c111b2ce51a)